api/v1/kubectlstorageosconfig_types.go

```diff
@@ -89,6 +89,8 @@ type Install struct {
 	PortalSecret                    string `json:"portalSecret,omitempty"`
 	PortalTenantID                  string `json:"portalTenantID,omitempty"`
 	PortalAPIURL                    string `json:"portalAPIURL,omitempty"`
+	InstallLocalPathProvisioner     bool   `json:"includeLocalPathProvisioner,omitempty"`
```

Comment:
Does it make sense for this to exist one level higher and to be `IncludeLocalPathProvisioner` so that it is common for bot install and uninstall (same as `IncludeEtcd`)